packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -77,6 +77,16 @@ const MANAGER = new CurlyComponentManager();
 import { ComponentDefinition, ValueReference } from 'glimmer-runtime';
 import Component from '../ember-views/component';
 
+function tagName(vm) {
+  let { tagName } = vm.dynamicScope().view;
+
+  if (tagName === '') {
+    throw new Error('Not implemented: fragments (`tagName: ""`)');
+  }
+
+  return new ValueReference(tagName || 'div');
```

Comment:
Kk, gotcha. For future travelers the current implementation is https://github.com/emberjs/ember.js/blob/master/packages/ember-views/lib/system/build-component-template.js#L136-L153